### PR TITLE
git-scp: add page

### PR DESCRIPTION
**Pull request by pg99696**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/git-scp.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# git scp
+
+> Copy files from the current working tree to the working directory of a remote repository.
+> Internally this script uses rsync and not scp as the name suggests.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-scp>.
+
+- Copy unstaged files to remote:
```
Comment (EmilySeville7cfg) [inline pages/common/git-scp.md:8]:
```suggestion
- Copy unstaged files to a remote:
```

Optional change: no such style guide requirement exists.

File: pages/common/git-scp.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+`git scp staging`
+
+- Copy staged and unstaged files to remote:
```
Comment (EmilySeville7cfg) [inline pages/common/git-scp.md:12]:
```suggestion
- Copy staged and unstaged files to a remote:
```

Optional change: no such style guide requirement exists.

File: pages/common/git-scp.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+`git scp staging HEAD`
+
+- Copy files that has been changed in the last commit, plus any staged or unstaged files to remote:
```
Comment (EmilySeville7cfg) [inline pages/common/git-scp.md:16]:
```suggestion
- Copy files that has been changed in the last commit and any staged or unstaged files to a remote:
```

Optional change: no such style guide requirement exists.

File: pages/common/git-scp.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+`git scp staging HEAD~1`
+
+- Copy specific files:
+
+`git scp staging {{path/to/file1}}...{{path/to/filen}}`
+
+- Copy specific directory:
```
Comment (EmilySeville7cfg) [inline pages/common/git-scp.md:24]:
```suggestion
- Copy a specific directory:
```

Optional change: no such style guide requirement exists.

**Review by EmilySeville7cfg — APPROVED** [review]
- PR was approved because there are just optional fixes can be applied.
- This page should be reviewed by `git-scp` users.
- Thanks for the contribution. 🚀

File: pages/common/git-scp.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# git scp
+
+> Copy files from the current working tree to the working directory of a remote repository.
+> Internally this script uses rsync and not scp as the name suggests.
```
Comment (kbdharun) [inline pages/common/git-scp.md:4]:
```suggestion
```
I don't think this line is necessary as we allow a maximum of 2 lines of page description and 1 for more information link although there are some exceptions. If you think this is an essential line apply the change below.

```suggestion
> Internally this script uses `rsync` and not `scp` as the name suggests.
```

File: pages/common/git-scp.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+`git scp staging HEAD`
+
+- Copy files that has been changed in the last commit, plus any staged or unstaged files to remote:
+
+`git scp staging HEAD~1`
+
+- Copy specific files:
+
+`git scp staging {{path/to/file1}}...{{path/to/filen}}`
```
Comment (kbdharun) [inline pages/common/git-scp.md:22]:
```suggestion
`git scp staging {{path/to/file1 file2 ... filen}}`
```
I guess this can be used here

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/git-scp.md (hunk 6 above)
Comment (pg99696) [reply pages/common/git-scp.md]: This can be n no. of files. Don't you think the previous syntax give more clarity.

Comment (pg99696) [reply pages/common/git-scp.md]: > There can be n no. of files. Also the path can different for different files. 
Don't you think the previous syntax give more clarity?

Comment (kbdharun) [reply pages/common/git-scp.md]: I am not sure, We generally use these multiple command arguments but we have been using it for files and directories too recently, It isn't documented in the style guide. So I will ask other's opinions.

cc @navarroaxel @EmilySeville7cfg

Comment (EmilySeville7cfg) [reply pages/common/git-scp.md]: If `...` is a mandatory syntax part of the command then we can leave the current code example as is without modification.

Comment (pg99696) [reply pages/common/git-scp.md]: > If `...` is a mandatory syntax part of the command then we can leave the current code example as is without modification. But at the end of the day @pg99696 should decide what is better based on our opinions. ;)

It's not mandatory, there can be 1 to N no. of files in the command. Please check this doc page for your reference. 
https://github.com/tj/git-extras/blob/master/Commands.md#git-scp

File: pages/common/git-scp.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# git scp
+
+> Copy files from the current working tree to the working directory of a remote repository.
+> Internally this script uses rsync and not scp as the name suggests.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-scp>.
+
+- Copy unstaged files to a remote:
+
+`git scp staging`
```
Comment (navarroaxel) [inline pages/common/git-scp.md:10]:
```suggestion
`git scp {{remote_name}}`
```

File: pages/common/git-scp.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
+# git scp
+
+> Copy files from the current working tree to the working directory of a remote repository.
+> Internally this script uses rsync and not scp as the name suggests.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-scp>.
+
+- Copy unstaged files to a remote:
```
Comment (navarroaxel) [inline pages/common/git-scp.md:8]:
```suggestion
- Copy unstaged files to a specific remote:
```

File: pages/common/git-scp.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+`git scp staging`
+
+- Copy staged and unstaged files to a remote:
+
+`git scp staging HEAD`
```
Comment (navarroaxel) [inline pages/common/git-scp.md:14]:
```suggestion
`git scp {{remote_name}} HEAD`
```

File: pages/common/git-scp.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+- Copy files that has been changed in the last commit and any staged or unstaged files to a remote:
+
+`git scp staging HEAD~1`
+
+- Copy specific files:
+
+`git scp staging {{path/to/file1}}...{{path/to/fileN}}`
```
Comment (navarroaxel) [inline pages/common/git-scp.md:22]:
```suggestion
`git scp {{remote_name}} {{path/to/file1 path/to/file2 ...}}`
```

File: pages/common/git-scp.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+
+- Copy staged and unstaged files to a remote:
+
+`git scp staging HEAD`
+
+- Copy files that has been changed in the last commit and any staged or unstaged files to a remote:
+
+`git scp staging HEAD~1`
+
+- Copy specific files:
```
Comment (navarroaxel) [inline pages/common/git-scp.md:20]:
```suggestion
- Copy specific files to a remote:
```

File: pages/common/git-scp.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+
+`git scp staging {{path/to/file1}}...{{path/to/fileN}}`
+
+- Copy a specific directory:
+
+`git scp staging {{path/to/directory}}`
```
Comment (navarroaxel) [inline pages/common/git-scp.md:26]:
```suggestion
`git scp {{remote_name}} {{path/to/directory}}`
```

File: pages/common/git-scp.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+
+- Copy files that has been changed in the last commit and any staged or unstaged files to a remote:
+
+`git scp staging HEAD~1`
+
+- Copy specific files:
+
+`git scp staging {{path/to/file1}}...{{path/to/fileN}}`
+
+- Copy a specific directory:
```
Comment (navarroaxel) [inline pages/common/git-scp.md:24]:
```suggestion
- Copy a specific directory to a remote:
```

File: pages/common/git-scp.md [hunk 14]
```diff
@@ -0,0 +1,26 @@
+# git scp
+
+> Copy files from the current working tree to the working directory of a remote repository.
+> Internally this script uses rsync and not scp as the name suggests.
+> Part of `git-extras`.
```
Comment (navarroaxel) [inline pages/common/git-scp.md:5]:
```suggestion
> Part of `git-extras`. Uses `rsync` to transfer files.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**